src/components/GitLink/GitRepoLink.tsx

```diff
@@ -35,7 +35,13 @@ const GitRepoLink: React.FC<React.PropsWithChildren<Props>> = ({
 
   return (
     <Tooltip content={fullUrl} position={TooltipPosition.bottom}>
-      <ExternalLink href={fullUrl} icon={icon} hideIcon dataTestID={dataTestID}>
+      <ExternalLink
+        href={fullUrl}
+        icon={icon}
+        hideIcon
+        dataTestID={dataTestID}
+        additionalClassName={'highlightable-link'}
```

Comment:
Add the classname in the External link component conditionally. You can use a boolean prop something like `isHighlightable` to add the className in the component itself. Consumer of component shouldn't have to worry about which class name is needed.

Reply:
Good idea, I applied the changes